scripts/perf-init.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+
+const execa = require('execa');
+const path = require('path');
+const fs = require('fs');
+const sander = require('sander');
+const repo = process.argv[2];
+
+const TARGET_DIR = path.resolve(__dirname, '..', 'perf');
+const VALID_REPO = /^[^/\s]+\/[^/\s]+$/;
+
+if (process.argv.length !== 3 || !VALID_REPO.test(repo)) {
+	console.error(
+		'You need to provide a GitHub repo in the form <username>/<repo>, e.g. ' +
+			'"npm run perf:init rollup/rollup"'
```

Comment:
Just thought I'd mention - it could be useful to support `npm run perf:init rollup/rollup#branch-name` where the branch becomes `-b ${branchName}` in the git clone when provided.

Reply:
Good idea! Even though the intention is to test is your current installation of rollup and not what you install in the `perf` folder, this feature could also be used to easily work on and debug rollup problems with branches in repos.